packages/react/src/primitives/SliderField/__tests__/SliderField.test.tsx

```diff
@@ -85,13 +86,43 @@ describe('SliderField: ', () => {
 
     it('should display value if isValueHidden is false', async () => {
       render(<SliderField defaultValue={10} label="slider" />);
-      const value = await screen.queryByText('10');
+      const value = screen.queryByText('10');
       expect(value).toBeInTheDocument();
     });
 
+    it('should display string formatted value if formatValue is provided', async () => {
```

Comment:
Nit: should we have a test for null, undefined, or other unhappy case values?

Reply:
Let me add a test to cover these 2 nil values

Reply:
The basic use test should kinda cover nil value, where by default the prop should be `undefined`